modules/learn/pages/services-and-indexes/services/backup-service.adoc

```diff
@@ -83,6 +83,14 @@ Optionally, the data itself can be retained, on the local filesystem.
 A deleted repository whose data still exists can be _imported_ back into the cluster, if required.
 Once imported, the repository can be _read_ from, but no longer receives data backups.
 
+[#avoiding-task-overlap]
+== Avoiding Task Overlap
+
+Although the Backup Service allows automated tasks to be scheduled for repetition at intervals as small as one minute, administrators are recommended typically not to lower the interval below fifteen minutes; and always to ensure that the interval is large enough to allow each scheduled task-repetition ample time to complete before the next is due to commence; even in the event of unanticipated network latency.
```

Comment:
I think `repetition` may be a bit confusing in some of the places I would suggest altering the following

> Backup Service allows automated tasks to be scheduled for repetition at intervals as small as one minute

To

> Backup Service allows automated task to be scheduled at intervals as small as one minutes

Reply:
The reason I used 'repetition' is that I imagined we are talking about the same task, scheduled to be repeated at intervals?.But perhaps that is not correct. If *different* tasks overlap, does this also produce failure of this kind?